clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -2777,12 +2777,16 @@ protected void validateMessage(
 
       // TODO: remove this condition check after fixing the bug that drainer in leaders is validating RT DIV info
       if (consumerRecord.getValue().producerMetadata.messageSequenceNumber != 1) {
-        LOGGER.warn(
-            "Data integrity validation problem with: {} offset: {}, "
-                + "but consumption will continue since EOP is already received. Msg: {}",
-            consumerRecord.getTopicPartition(),
-            consumerRecord.getOffset(),
-            warningException.getMessage());
+        String errorMsgIdentifier = consumerRecord.getTopicPartition().getPubSubTopic().getName() + "-"
```

Comment:
but warnMessage should contain details of producer right? Should we ignore that?

Reply:
NVM. Didn't see the getClass().getName() this should be OK.